style.css

```diff
@@ -0,0 +1,566 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Jua", sans-serif;
+}
+
+.transform {
+  transform: rotate(-11deg);
+  box-shadow: 10px 20px 10px -15px black;
+}
+
+header,
+html {
+  width: 100%;
+}
+
+h2 {
+  display: none;
+}
+
+footer div {
+  margin: 0 40px 0 40px;
+}
+
+footer h4 {
+  font-size: 20px;
+  color: #fff;
+  padding-top: 35px;
+  margin-bottom: 35px;
+}
+
+footer > div > ul > a {
+  text-decoration: none;
+  color: #000;
+}
+
+nav {
+  background-color: gray;
+  width: 100%;
+  height: 175px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 15px;
+}
+
+.logo {
+  background-color: #696969;
+  width: 210px;
+  height: 76px;
+  margin: 5px 15px;
+  position: relative;
+}
+
+.navbar {
+  text-align: center;
+}
+
+.navbar > a {
+  margin: 0 auto;
+  color: rgb(255, 248, 248);
```

Comment:
- [ ] [ OPTIONAL] this project should be made all in tones of grey.
The way you make a shade of grey in rgb is by using **the same number for the three arguments**, the way you can make a grey using hex colors is by using the same string three times, ex: #a1a1a1.
A lot of the 'greys' you are using still have color in them. Try to apply the rules mentioned above to all the places you set color and background color. 
The reason for that is that the heatmap is about showing the visual weight of each visual element of the page and when you have color near black and whites and greys, even if it's very muted, it calls a lot of visual weight to itself, thus defeating the objective of the heat map.